test/common_utils.py

```diff
@@ -339,13 +339,13 @@ def freeze_rng_state():
 class TransformsTester(unittest.TestCase):
 
     def _create_data(self, height=3, width=3, channels=3, device="cpu"):
-        tensor = torch.randint(0, 255, (channels, height, width), dtype=torch.uint8, device=device)
+        tensor = torch.randint(0, 256, (channels, height, width), dtype=torch.uint8, device=device)
```

Comment:
`randint` is exclusive on the upper bound, so the right value here is 256. I opted in fixing this small bug in-place to avoid doing unnecessary CI runs.